src/directives/for-of/for_of.directive.ts

```diff
@@ -404,6 +410,17 @@ export class IgxForOfDirective<T> implements OnInit, OnChanges, DoCheck, OnDestr
         }
     }
 
+    protected _calcMaxBrowserHeight() {
```

Comment:
@MayaKirova The new functions start with `_calc`(_calcMaxBrowserHeight, _calcHeight). The old one is `_calculate` (_calculateChunkSize). Maybe we should adopt a unified prefix for `_calc`.
There is no return type declared.